connect/transforms/src/main/java/org/apache/kafka/connect/transforms/ExtractField.java

```diff
@@ -21,13 +21,16 @@
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.Struct;
 import org.apache.kafka.connect.transforms.util.SimpleConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.Map;
 
 import static org.apache.kafka.connect.transforms.util.Requirements.requireMapOrNull;
 import static org.apache.kafka.connect.transforms.util.Requirements.requireStructOrNull;
 
 public abstract class ExtractField<R extends ConnectRecord<R>> implements Transformation<R> {
+    private static final Logger log = LoggerFactory.getLogger(ExtractField.class);
```

Comment:
I'm not sure any of these log messages add value since TransformationChain has log messages with the records, which should be sufficient to know whether the schemas are null.